kogito-usertasks-with-security-oidc-keycloak/README.md

```diff
@@ -0,0 +1,211 @@
+# Kogito user task orchestration
+
+## Description
+
+A quickstart project shows very typical user task orchestration. It comes with two tasks assigned
+to human actors via groups assignments - `managers`. So essentially anyone who is a member of that 
+group can act on the tasks. Though this example applies four eye principle which essentially means 
+that user who approved first task cannot approve second one. So there must be always at least two 
+distinct manager involved.
+
+This example shows
+
+* working with user tasks
+* four eye principle with user tasks
+	
+	
+<p align="center"><img width=75% height=50% src="docs/images/process.png"></p>
+
+
+## Build and run
+
+### Prerequisites
+ 
+You will need:
+  - Java 1.8.0+ installed 
+  - Environment variable JAVA_HOME set accordingly
+  - Maven 3.5.4+ installed
+  - [jq](https://stedolan.github.io/jq) tool installed. You can download it from [here](https://stedolan.github.io/jq/download)
+  - Docker
+
+
+When using native image compilation, you will also need: 
+  - GraalVM 19.1+ installed
+  - Environment variable GRAALVM_HOME set accordingly
+  - Note that GraalVM native image compilation typically requires other packages (glibc-devel, zlib-devel and gcc) to be installed too, please refer to GraalVM installation documentation for more details.
+
+###Starting and Configuring the Keycloak Server
```

Comment:
missing space after ### so it renders without making it a title

Reply:
ups, updated